api/envoy/extensions/filters/http/credential_injector/v3alpha/credential_injector.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.credentials.v3alpha;
+
+import "envoy/config/core/v3/http_uri.proto";
+import "envoy/extensions/transport_sockets/tls/v3/secret.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.credential_injector.v3alpha";
+option java_outer_classname = "CredentialInjectorProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/http/credential_injector/v3alpha;credential_injectorv3alpha";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Credentials Injector]
+// Credentials Injector :ref:`configuration overview <config_http_filters_credential_injector>`.
```

Comment:
I think you want "not-implemented-hide" on this proto since this reference is not defined yet. This should probably fix the CI failure.

Reply:
yep - its complaining because its not added to either of the `...build_config.bzl` files